packages/test/test-service-load/src/workloads/gc/testConfig.json

```diff
@@ -79,18 +75,16 @@
 					}
 				},
 				"odsp": {
-					"comment": "SessionExpiry: 15 mins. SnapshotCacheExpiry: 5 mins. Inactive Timeout: 25 mins. Sweep Timeout: 26 mins.",
+					"comment": "SessionExpiry: 15 mins. SnapshotCacheExpiry: 5 mins. Inactive / Sweep Timeout: 25 mins.",
```

Comment:
grace period

Reply:
Missing on some others as well

Reply:
I wonder if the GC tests should have some default options hardcoded for all profiles (different from the production defaults) and then they can be adjusted if needed.  To make it more obvious where the profiles are the same v. different and avoid mistakes where a profile isn't updated but should be.

Reply:
This is for tombstone mode. See profile name "ci_tombstone". Sweep does not run in this mode. With two staged sweep, maybe we can get rid of the two and only have one config which runs both?

Reply:
> I wonder if the GC tests should have some default options hardcoded for all profiles (different from the production defaults) and then they can be adjusted if needed. To make it more obvious where the profiles are the same v. different and avoid mistakes where a profile isn't updated but should be.

I am not sure I follow where the default would be? In options matrix?